src/cls/rgw/cls_rgw.cc

```diff
@@ -2168,8 +2168,6 @@ int rgw_dir_suggest_changes(cls_method_context_t hctx,
   CLS_LOG(1, "entered %s", __func__);
 
   const ConfigProxy& conf = cls_get_config(hctx);
-  uint64_t max_objs = conf.get_val<uint64_t>("rgw_max_objs_per_shard");
-  CLS_LOG(0, "ERIC: rgw_max_objs_per_shard is %lu", max_objs);
```

Comment:
naughty

Reply:
That was my initial test to make sure I could access a config. It should have cancelled out an earlier commit. (Final step will be to consolidate commits.)